elastic/kibana
# [Uptime] monitor management - adjust deletion logic

## Summary

Resolves https://github.com/elastic/kibana/issues/146932

Adjusts monitor delete logic for Uptime to ensure that multiple monitors are able to be deleted in a row. 

### Testing
1. Create at least two monitors
2. Navigate to Uptime monitor management. Delete a monitor. Ensure the success toast appears and the monitor is removed from the monitor list
3. Delete a second monitor. Ensure the success toast appears and the monitor is removed from the list.
